shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -340,11 +340,6 @@ - (void)setupNotificationCenterObservers {
                  name:UIKeyboardWillHideNotification
                object:nil];
 
-  [center addObserver:self
-             selector:@selector(onAccessibilityStatusChanged:)
-                 name:UIAccessibilityVoiceOverStatusChanged
```

Comment:
Don't we need to replace this with `UIAccessibilityVoiceOverStatusDidChangeNotification` instead of removing it?

Reply:
Oh, I mistakenly thought the below notification is `UIAccessibilityVoiceOverStatusDidChangeNotification` and said we have already implemented it in the issue. I updated the issue and fixed this.